servarr/values.yaml

```diff
@@ -28,12 +28,50 @@ tags:
   # @default -- true
   # @section -- Tags
   music: true
+  # -- (bool) This tag will deploy: Scraparr
+  # @default false
+  # @section -- Tags
+  metrics: false
```

Comment:
I think this is not required and can be also confusional. Instead we can introduce (and use) `scraparr.enabled`